raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2755,7 +2755,7 @@ private FetchRequestData buildFetchRequest() {
             .setReplicaState(new FetchRequestData.ReplicaState().setReplicaId(quorum.localIdOrSentinel()));
     }
 
-    private long maybeSendAnyBootstrapFetch(long currentTimeMs) {
+    private long maybeSendToAnyBootstrap(long currentTimeMs) {
```

Comment:
How about `maybeSendFetchToAnyBootstrap`? I see that this is the name I suggested. I forgot to also include the word "fetch".